modules/kmm-sign-validation-stage.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * updating/kmm-preflight-validation.adoc
+
+:_content-type: CONCEPT
+[id="kmm-sign-validation-stage_{context}"]
+= Sign validation stage
+
+Sign validation is executed only when image validation has failed, there is a `sign` section in the `Module` that is relevant for the upgrade kernel, and build validation finished successfully in the event there was a `build` section in the `Module` relevant for the upgraded kernel.
+Sign validation will try to run the sign job and validate that it finishes successfully.
+
+If the `PushBuiltImage` flag is defined in the `PreflightValidationOCP` CR, sign validation will also try to
+push the resulting image to its registry.
+
+The resulting image is always the image defined in the `ContainerImage` field of the `Module`. The input image is either the output of the Build stage, or an image defined in the `UnsignedImage` field.
```

Comment:
Should it be lowercase "c" `containerImage` like the others?

Reply:
yes. changed.